estate/models/estate_property.py

```diff
@@ -0,0 +1,35 @@
+import datetime as dt
+
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "An awesome estate module"
```

Comment:
```suggestion
    _description = "Estate Property"
```
The description should describe the model only, not the whole module 😉